### PR TITLE
Updates per revised guidance on adding a copilot section to docs

### PR DESCRIPTION
**Pull request by anandmeg**
---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/csharp/how-to/parse-strings-using-split.md](https://github.com/dotnet/docs/blob/4e35fb40245eacf33c769bd7fcaaa20ff7e09026/docs/csharp/how-to/parse-strings-using-split.md) | [How to separate strings using String.Split in C\#](https://review.learn.microsoft.com/en-us/dotnet/csharp/how-to/parse-strings-using-split?branch=pr-en-us-43994) |
| [docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md](https://github.com/dotnet/docs/blob/4e35fb40245eacf33c769bd7fcaaa20ff7e09026/docs/csharp/programming-guide/types/how-to-convert-a-string-to-a-number.md) | ["How to convert a string to a number"](https://review.learn.microsoft.com/en-us/dotnet/csharp/programming-guide/types/how-to-convert-a-string-to-a-number?branch=pr-en-us-43994) |

### CODE REVIEW

File: docs/csharp/how-to/parse-strings-using-split.md [hunk 1]
```diff
@@ -49,64 +52,17 @@ Consecutive instances of any separator produce the empty string in the output ar
...
-
-**Example Output**
-You
-win
-some
-You
-lose
-some
+```copilot-prompt
+Generate C# code to use Split.String split a string into substrings. Input string is "You win some. You lose some." Delimiters are space and period.  Provide example output.
```
Comment (gewarren) [inline docs/csharp/how-to/parse-strings-using-split.md:60]:
```suggestion
Generate C# code to use Split.String to split a string into substrings. Input string is "You win some. You lose some." Delimiters are space and period. Provide example output.
```

**Review by gewarren — COMMENTED** [review]
There are also a couple markdownlint errors to fix (multiple blank lines).

**Review by gewarren — APPROVED** [review]

File: docs/csharp/how-to/parse-strings-using-split.md [hunk 2]
```diff
@@ -49,64 +52,19 @@ Consecutive instances of any separator produce the empty string in the output ar
...
-}
-
-**Example Output**
-You
-win
-some
-You
-lose
-some
+```copilot-prompt
```
Comment (gewarren) [inline docs/csharp/how-to/parse-strings-using-split.md:59]: Is `copilot-prompt` a supported language tag?

Comment (anandmeg) [reply docs/csharp/how-to/parse-strings-using-split.md:59]: Yes. https://review.learn.microsoft.com/en-us/help/platform/metadata-fields/ms-devlang?branch=main

Comment (anandmeg) [reply docs/csharp/how-to/parse-strings-using-split.md:59]: copilot-prompt was added recently to enable adding copilot prompts to docs in this way.

**Review by BillWagner — APPROVED** [review]
This LGTM @anandmeg 

I'll :shipit:

**Pull request merged**